src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/cpp/CppCompletionRequest.java

```diff
@@ -392,6 +399,18 @@ private void applyValue(CppCompletion completion)
          else
             insertText = insertText + "(";
       }
+      else if (completion.getType() == CppCompletion.DIRECTORY)
+      {
+         insertText += "/";
+         Scheduler.get().scheduleDeferred(new ScheduledCommand()
+         {
+            @Override
+            public void execute()
+            {
+               events_.fireEvent(new RequestCppCompletionsEvent());
```

Comment:
The goal here is to automatically pop up a new set of completions after inserting a directory completion -- is there a cleaner way?